kinder/pkg/cluster/manager/actions/waiter.go

```diff
@@ -255,6 +267,33 @@ func podsAreRunning(n *status.Node, label string, replicas int) func(c *status.C
 	}
 }
 
+// nodePortIsReady implements a function that tests if a nodePort is ready
+func nodePortIsReady(n *status.Node, port string) func(c *status.Cluster, n *status.Node) bool {
+	return func(c *status.Cluster, n *status.Node) bool {
+		fmt.Printf("checking node port %s on node %s...\n", port, n.Name())
```

Comment:
i think we should remove this call.
and only leave the call on line 289. it prints only on a positive outcome (return true)

Reply:
okay 👍